data/news.yaml

```diff
@@ -1,7 +1,12 @@
-- content: "Join us on Tuesday, April 29th, 5pm GMT/ 6pm CET / 12pm CDT / 10am PDT, when Brittany Istenes, InnerSource Commons Member, will discuss [Empathetic Engineering and InnerSource](https://bit.ly/ISCApr29)."
+- content: "Join us on Tuesday, May 6th, 9am GMT / 10am CEST / 1:30 IST / 4pm CST / 6pm AEST, when Jerry Tan, from COPU, will discuss
+ [InnerSource in China - Past, Present, and Future.](https://bit.ly/ISCMay06)."
   active: true
   type: "primary"
 
+- content: "Join us on Tuesday, April 29th, 5pm GMT/ 6pm CET / 12pm CDT / 10am PDT, when Brittany Istenes, InnerSource Commons Member, will discuss [Empathetic Engineering and InnerSource](https://bit.ly/ISCApr29)."
```

Comment:
Can remove this now?

Reply:
No, we don't need to do that if we set the parameter `active: false`